### ISSUE TITLE
motion forecasting corresponding lidar data

### ISSUE THREAD

**Issue by BaiLiping** [argoai/av2-api#4]
hi, if i want to do motion forecasting from lidar data, can i find the corresponding lidar raw data for each frame?

**Comment by James-Hays** [comment]
Hi!

Unfortunately, no. The lidar data for 250,000 motion forecasting scenarios spanning 763 hours would be about 25 TB with moderate compression.

However, the lidar dataset is 166 hours total, roughly 1/5th the time span of the motion forecasting dataset. It doesn't have any notion of ground truth trajectories, but you could run your own tracker to create trajectories.

**Comment by BaiLiping** [comment]
thanks for your reply. I think i will use that 166h of data then. But how do i do the evaluation on that 166h of forcasting based on the raw lidar data?

**Comment by James-Hays** [comment]
If you're trying to forecast objects, you'd need a tracker to give you some notion of ground truth tracks to evaluate against. We don't have "official" tracks to provide. Although if the community is interested in such tracks that would be good to know.

Alternatively, you could try to forecast the point cloud itself (e.g. SPF2 https://www.xinshuoweng.com/papers/SPF2/proceeding.pdf ).

**Comment by James-Hays** [comment]
I should add that the Argoverse 2 Sensor Dataset has all sensors as well as ground truth tracks (for the training and validation set), but it is only 4.2 hours of data. It is similar in scale to nuScenes or Waymo open's sensor datasets.

**Comment by BaiLiping** [comment]
Thanks a lot for the pointers. I would look into those.
